assets/styles/prosilver/content.css

```diff
@@ -854,6 +854,7 @@ table.fixed-width-table {
 .postbody .content::-webkit-scrollbar, .topicreview::-webkit-scrollbar, .post_details::-webkit-scrollbar, .codebox code::-webkit-scrollbar, .attachbox dd::-webkit-scrollbar, .attach-image::-webkit-scrollbar, .dropdown-extended ul::-webkit-scrollbar {
 	width: 8px;
 	height: 8px;
+	appearance: none;
```

Comment:
Whats this for? Would be good if you could remove it.

Reply:
This is to avoid error messages, actually it always gives all commands for the different browsers, here only the command for firefox is given.